lib/policial/client.rb

```diff
@@ -0,0 +1,41 @@
+module Policial
+  # Public: Starting with an Octokit client and a pull request,
+  # it checks all changes introduced looking for style guide violations. It
+  # also accuses all present violations through comments on all relevant lines
+  # in the pull request.
+  class Client
```

Comment:
This classes could be renamed to something else to better intention revealing. This `Client` class is not just a client. It knows how to investigate a pull request and know about violations and know what accuse. Maybe `Policial::Detective`?